interface/billing/sl_receipts_report.php

```diff
@@ -413,38 +433,49 @@ function sel_diagnosis() {
       // If a doctor was specified.
       if ($form_doctor) $query .= " AND u.id = '$form_doctor'";
       ***************************************************************/
+      $sqlBindArray = array();
       $query = "SELECT a.pid, a.encounter, a.post_time, a.code, a.modifier, a.pay_amount, " .
         "fe.date, fe.id AS trans_id, fe.provider_id AS docid, fe.invoice_refno, s.deposit_date, s.payer_id, " .
-        "b.provider_id " .
+        "b.provider_id, concat(lname, ' ', fname) as 'pat_fulname' " .
```

Comment:
Need to make it clear what table the lname and fname are coming from since this query operates on a lot of tables:
``````
"b.provider_id, concat(p.lname, ' ', p.fname) as 'pat_fulname' " .